ark/analysis/visualize.py

```diff
@@ -103,9 +170,11 @@ def draw_heatmap(data, x_labels, y_labels, dpi=None, center_val=None,
     sns.set(font_scale=.7)
 
     if overlay_values:
-        sns.clustermap(data_df, cmap=colormap, annot=data, center=center_val)
+        sns.clustermap(data_df, cmap=colormap, annot=data, center=center_val,
```

Comment:
@ngreenwald good point, this might be residual code from CIMAC. I'll have to see if this is used anywhere else in the function, but otherwise I'll remove.